docs/ref/contrib/admin/index.txt

```diff
@@ -1076,6 +1077,55 @@ subclass::
     Don't include a field in ``radio_fields`` unless it's a ``ForeignKey`` or has
     ``choices`` set.
 
+.. attribute:: ModelAdmin.autocomplete_fields
+
+    .. versionadded:: 1.11
+
+    By default, Django's admin uses a select-box interface (<select>) for
+    fields that are ``ForeignKey`` or ``ManyToManyField``.
+    Sometimes you don't want to incur the overhead of having to select all
+    the related instances to display in the drop-down.
+
+    ``autocomplete_fields`` is a list of fields that you would like to change
+    to `Select2 <contrib-admin-select2>`_ autocomplete inputs.
+
+    The Select2 input looks similar to the default input, but comes with a
+    search feature and loads the options asynchronous. This has advantages
+    if you target table has many rows.
+
+    The autocomplete search is performed based on the
+    :attr:`ModelAdmin.search_fields` defined on the related object.
+    Therefore it is required to define :attr:`ModelAdmin.search_fields`
+    on the related ``ModelAdmin``.
+
+    The ordering is defined by the related ``ModelAdmin``'s
+    :attr:`ModelAdmin.ordering` implementation.
+
+    Example::
+
+        class QuestionAdmin(admin.ModelAdmin):
+            search_fields = ('question_text',)
+            ordering = ('date_created',)
+
+        class ChoiceAdmin(admin.ModelAdmin):
+            autocomplete_fields = ('question',)
+
+    In the example, the ``ChoiceAdmin`` will have an autocomplete field for the
+    ``ForeignKey`` to the ``Question``. The results will be filtered by the
+    ``question_text`` and sorted by the ``date_created`` field.
+
+    .. admonition:: Note
```

Comment:
I guess you misinterpreted my previous comment. "Note" isn't a very descriptive title. The idea is to give an overview of what the box describes to help skimming.